apps/silverback-drupal/web/modules/custom/silverback_gatsby_test/src/Plugin/GraphQL/Schema/SilverbackGatsbyTestSchema.php

```diff
@@ -93,6 +93,19 @@ public function getResolverRegistry(): ResolverRegistryInterface {
       $builder->produce('url_path')->map('url', $builder->fromParent()),
     ));
 
+    $addResolver('Webform.path',
+      $builder->compose(
+        $builder->produce('entity_url')->map('entity', $builder->fromParent()),
+        $builder->produce('url_path')->map('url', $builder->fromParent())
+      )
```

Comment:
This is repeated _a lot_. Wouldn't it make sense to also auto-create these resolvers when we have a `@path` directive on an `@entity` type?

Reply:
Yeah, `@isPath` and `@producePath` IIRC what we have discussed.